TimeConversion.java

```diff
@@ -0,0 +1,35 @@
+import java.util.Scanner;
+
+/*
+	https://www.hackerrank.com/challenges/time-conversion
+*/
+
+public class TimeConversion {
+
+	public static void main(String[] args) {
+		Scanner s=new Scanner(System.in);
+    	String st=s.next();
+		int i=Integer.parseInt(st.substring(0, 2));
+		if(st.charAt(8)=='P')
+		{
+			if(i!=12)
+			{
+			i+=12;
+			}
+			System.out.println(i+st.substring(2, 8));

```

Comment:
You repeated the st.substring(2, 8) 3 times in this code. Do it only once at the top and store it in a varible.